redirect/routes.php

```diff
@@ -233,15 +233,19 @@ class Route {
         'statista' => [
             'description' => 'Statista',
             'target'      => 'https://de-statista-com.eaccess.ub.tum.de',
-        ]
-
+        ],
+        'sharelatex' => [
+            'description' => 'ShareLaTeX@TUM',
+            'target'      => 'https://sharelatex.tum.de/',
+        ],
     ];
 
     private $synonyms = [
         'erapra' => 'erapraktikum',
         'ma-sp'  => 'sp-ma',
         'eidi'  =>  'info1',
         'eidi2' =>  'info2',
+        'sharelatex' =>  'latex',
```

Comment:
This should be other way round `source=>target` - maybe add a comment?

Reply:
Ah, I see how I was using them wrong 🙈 didn't think much about it^^